can_tools/scrapers/official/federal/CDC/cdc_testing_cases.py

```diff
@@ -0,0 +1,60 @@
+from typing import Dict
+
+import pandas as pd
+
+from can_tools.scrapers.base import CMU
+from can_tools.scrapers.official.base import FederalDashboard
+
+# Set the maximum number of records to ten million to ensure we collect all rows in the dataset.
+# Currently the size of the dataset is 2 million records, so this may need to be updated (increased)
+# in the future
+SODA_API_RESPONSE_LIMIT = 10000000
+
+
+class CDCHistoricalTestingDataset(FederalDashboard):
+    has_location = True
+    location_type = "county"
+    source = "https://data.cdc.gov/Public-Health-Surveillance/United-States-COVID-19-County-Level-of-Community-T/nra9-vzzn/data"
+    source_name = "Centers for Disease Control and Prevention"
+    provider = "cdc2"
```

Comment:
I used cdc2 here (as with the historical vaccine data) to not mix up this data with the previously collected CDC Covid Tracker data. 

I think that it would be okay to have this data override the old CDC data, but I'm concerned that since the new dataset and the old CDC Tracker will have different `source`/`source_name` columns, I think we will end up with rows that are entirely duplicated except for the `source` and `source_name` columns, and I'm not sure how the pipeline would handle that

Reply:
If we _were_ to use the `cdc` provider here (and did not run into issues with duplicate entries) then I don't think we would need to change the downstream pipeline at all

Reply:
Hrm.  What's the primary key for rows in the database?  I would assume that if we used `cdc` then we'd just overwrite existing rows.  I guess the main risk would be if the new dataset had fewer historical datapoints than the old data.  We'd have a mix of sources in the resulting data, depending on the date?  In that case, the website might end up linking to the wrong place, I guess.  And the bigger issue would be that we might have jumps in the data if the two datasets aren't compatible.  To be safe we would delete the old data.

That said, per my other comment I think we may need to keep the old scraper and so making this be `cdc2` makes sense.  But let's add a comment to try to explain a bit of context.

Reply:
good question on the primary key 😅 I'm not actually sure -- I'll take a look